demos/dialog.html

```diff
@@ -0,0 +1,160 @@
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      https://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License
+-->
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>MDC-Web Dialog Demo</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500" rel="stylesheet">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <script src="assets/material-components-web.css.js" charset="utf-8"></script>
+    <style>
+    	.demo-body {
+    	  padding: 24px;
+    	  margin: 0;
+    	  box-sizing: border-box;
+    	}
+
+      .demo-content > button {
+        margin-bottom: 6px;
+      }
+    </style>
+  </head>
+  <body class="mdc-typography"> 
+    <div class="demo-body">
+    	<aside id="mdc-dialog-default"
+    	  class="mdc-dialog"
+    	  role="alertdialog"
+    	  aria-hidden="true"
+    	  aria-labelledby="mdc-dialog-default-label"
+    	  aria-describedby="mdc-dialog-default-description">
+    	  <div class="mdc-dialog__surface">
+    	    <header class="mdc-dialog__header">
+    	      <h2 id="mdc-dialog-default-label" class="mdc-dialog__header__title">
+    	        Use Google's location service?
+    	      </h2>
+    	    </header>
+    	    <section id="mdc-dialog-default-description" class="mdc-dialog__body">
+    	      Let Google help apps determine location. This means sending anonymous location data to Google, even when no apps are running.
+    	    </section>
+    	    <footer class="mdc-dialog__footer">
+    	      <button type="button" class="mdc-button mdc-dialog__footer__button mdc-dialog__footer__button--cancel">decline</button>
+    	      <button type="button" class="mdc-button mdc-dialog__footer__button mdc-dialog__footer__button--accept">accept</button>
+    	    </footer>
+    	  </div>
+    	  <div class="mdc-dialog__backdrop"></div>
+    	</aside>
+
+    	<aside id="mdc-dialog-with-list"
+    	  class="mdc-dialog"
+    	  role="alertdialog"
+    	  aria-hidden="true"
+    	  aria-labelledby="mdc-dialog-with-list-label"
+    	  aria-describedby="mdc-dialog-with-list-description">
+    	  <div class="mdc-dialog__surface">
+    	    <header class="mdc-dialog__header">
+    	      <h2 id="mdc-dialog-with-list-label" class="mdc-dialog__header__title">
+    	        Choose a Ringtone
+    	      </h2>
+    	    </header>
+    	    <section id="mdc-dialog-with-list-description" class="mdc-dialog__body mdc-dialog__body--scrollable">
+    	     	<ul class="mdc-list">
```

Comment:
indentation again 😄 . Please double-check both your HTML indentation as well as ensure that opening and closing tags before pushing up the next PR update.